app/__init__.py

```diff
@@ -45,21 +44,3 @@ def create_app(config_name):
     application.register_blueprint(status_blueprint)
 
     return application
-
-
-def isolation_level(level):
-    """Return a Flask view decorator to set SQLAlchemy isolation level
-
-    Usage::
-        @view("/thingy/<id>", methods=["POST"])
-        @isolation_level("SERIALIZABLE")
-        def create_thing(id):
-            ...
-    """
-    def decorator(view):
-        @wraps(view)
-        def view_wrapper(*args, **kwargs):
-            db.session.connection(execution_options={'isolation_level': level})
-            return view(*args, **kwargs)
-        return view_wrapper
-    return decorator
```

Comment:
I... think we should keep this. It's debugged, it works, it's not like we're never going to need to set an alternate serialization level.

Reply:
If this PR goes in it's not used anywhere, and if we need it in the future it's in the commit history?

Reply:
Nobody ever looks in the commit history - it'll be as good as gone and before you know it the next person who needs this will have done something godawful like injecting a manual `SET ...` statement in the beginning of a view or ... *something*

Reply:
Fair enough. Have added un-removed it.